deploy/rbac-permissions-operator-config/60-dedicated-admin-openshift-namespaces.SubjectPermission.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: dedicated-admins
+  namespace: openshift-rbac-permissions
+spec:
+  subjectKind: Group
+  subjectName: dedicated-admins
+  clusterPermissions:
+    - dedicated-admins-cluster
+  permissions:
+    -
+      clusterRoleName: dedicated-admins-project
+      namespacesAllowedRegex: "(^openshift-operators$|^openshift-operators-redhat$)"
+      namespacesDeniedRegex: ""
```

Comment:
```suggestion
```
We are not using denied.. is it a required property?  If not, no sense keeping it.